classes/student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
```

Comment:
- [ ] **[OPTIONAL]** Based on ruby conventions, when importing modules or files that are part of the working directory, we use `require_relative`, instead of `require`. The `require` method is often reserved for external code, like gems and so on.

Reply:
Done.